dist/performance.js

```diff
@@ -84,6 +84,66 @@ function getWebVitalsJS() {
     }).map(har => har.url);
 }
 
+function getGamingMetrics(rawDoc) {
+    let returnObj = {};
+    const regexForCheckChromeLH = new RegExp(/.{1}userAgent.{1,100}(?:Chrome-Lighthouse|Google Lighthouse).{2}/)
+    const regexForCheckGTmetrix = new RegExp(/.{1}userAgent.{1,100}(?:GTmetrix|gtmetrix.com).{2}/)
+    const regexForCheckPageSpeed = new RegExp(/.{1}userAgent.{1,100}(?:PageSpeed).{2}/)
+    let scripts = rawDoc.getElementsByTagName('script');
+    for (let i = 0; i < scripts.length; i++) {
+        if (scripts[i].src) {
+            continue;
+        }
+        let scriptTagCode = scripts[i].innerHTML;
+        if (regexForCheckChromeLH.test(scriptTagCode)){
+            returnObj['detectUA-ChromeLH'] = true;
+        }
+        if(regexForCheckGTmetrix.test(scriptTagCode)){
+            returnObj['detectUA-GTmetrix'] = true;
+        }
+        if(regexForCheckPageSpeed.test(scriptTagCode)){
+            returnObj['detectUA-PageSpeed'] = true;
+        }
+    }
+
+    //https://www.debugbear.com/blog/optimizing-web-vitals-without-improving-performance
+    //catch lcp animation / cls animation & overlay hack
+    const regexForCheckfadeInAnimation = new RegExp(/this.style.animation.{1,10}.fadein.{1,20}.forwards/)
+    let elements = rawDoc.getElementsByTagName('img');
+    for (let i = 0; i < elements.length; i++) {
+        let el = elements[i];
+        let onloadVal = el.getAttribute('onload');
+        if (onloadVal !== null) {
+            if(regexForCheckfadeInAnimation.test(onloadVal)) {
+                returnObj['lcpAnimation'] = true;
+            }
+        }
+        let styleObj = el.style;
+        if (styleObj['pointer-events'] == 'none' &&
```

Comment:
@mel-ada similarly, do you want to look at more general heuristics?

Like, if the image's area is more than 90% of the viewport and has `pointer-events: none`, it's probably an overlay hack.

Reply:
Yep, let's keep the current one and add something like: 
`lcpImgOverlaySoft = img > 90% of viewport && pointer-events: none`